docs/Bundle-Sizes.md

```
@@ -0,0 +1,34 @@
+| Data | `@artisanpack-ui/vue/data` | 26.6 KB | 6.1 KB | 3 components |
+| Feedback | `@artisanpack-ui/vue/feedback` | 22.3 KB | 4.3 KB | 6 components |
+| Utility | `@artisanpack-ui/vue/utility` | 24.0 KB | 5.7 KB | 5 components |
+| Composables | `@artisanpack-ui/vue/composables` | 3.8 KB | 1.6 KB | 2 composables |
+| **Full bundle** | `@artisanpack-ui/vue` | **333.9 KB** | **60.1 KB** | **56 components + 2 composables** |
+
+Sizes measured from the `dist/` output after `npm run build` (tsup with esbuild). Gzipped sizes use standard gzip compression. Vue and peer dependencies (apexcharts, vue3-apexcharts) are externalized and not included in these sizes.
+
+## Recommendations
+
+- **Import by category** instead of from the main entry point to minimize bundle size:
+  ```ts
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)